### PR TITLE
Issues 16, 17, 25, 67. Ready for beta release...

### PR DESCRIPTION
**Pull request by whiteluna**
https://github.com/acsylla/acsylla/issues/16
https://github.com/acsylla/acsylla/issues/17
https://github.com/acsylla/acsylla/issues/25
https://github.com/acsylla/acsylla/issues/67

Implemented all configuration options for Cluster, Batch and Statement

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,1221 @@
...
+
+Example for build wheel for Python 3.11 aarch64 from master branch
+```bash
+curl -O https://raw.githubusercontent.com/acsylla/acsylla/master/bin/build.sh
+curl -O https://raw.githubusercontent.com/acsylla/acsylla/master/bin/build_in_docker.sh
+chmod +x build.sh build_in_docker.sh
+./build_in_docker.sh 3.11 master aarch64
+```
+
+## Cluster
```
Comment (pfreixes) [inline README.md:109]: Any chance to move all deep documentation under its own `doc/documentation.md` file and leave into the Readme a basic snippet?

File: tests/test_statement.py [hunk 2]
```diff
@@ -65,6 +63,41 @@ async def test_create_with_execution_profile(self):
...
+
+    async def test_set_tracing(self, statement):
+        statement.set_tracing(True)
+        statement.set_tracing(False)
+        statement.set_tracing(None)
+
+    async def test_set_host(self, statement):
+        statement.set_host("127.0.0.1")
+        statement.set_host("127.0.0.1", port=123)
+        statement.set_host(None)
```
Comment (pfreixes) [inline tests/test_statement.py:99]: nice!

File: tests/test_batch.py [hunk 3]
```diff
@@ -61,3 +65,60 @@ def test_create_batch_counter_with_timeout_and_profile(self):
...
+        batch.set_retry_policy("default")
+        batch.set_retry_policy("fallthrough")
+        batch.set_retry_policy("default", retry_policy_logging=True)
+        batch.set_retry_policy("fallthrough", retry_policy_logging=True)
+        batch.set_retry_policy(None)
+
+    async def test_set_tracing(self, batch):
+        batch.set_tracing(True)
+        batch.set_tracing(False)
+        batch.set_tracing(None)
```
Comment (pfreixes) [inline tests/test_batch.py:124]: hats off!

**Review by pfreixes — APPROVED** [review]
Amazing work

File: README.md (hunk 1 above)
Comment (whiteluna) [reply README.md:109]: Yes of course when we add publishing docs to readthedocs

File: tests/test_batch.py (hunk 3 above)
Comment (whiteluna) [reply tests/test_batch.py:124]: @pfreixes Thanks! What do you think about bump version to 1.0.0 beta?

**Pull request merged**